subworkflows/nf-core/bam_variant_demix_boot_freyja/meta.yml

```diff
@@ -0,0 +1,76 @@
+name: "bam_variant_demix_boot_freyja"
+description: Recover relative lineage abundances from mixed SARS-CoV-2 samples from a sequencing dataset (BAM aligned to the Hu-1 reference)
+keywords:
+  - bam
+  - variants
+  - cram
+
+modules:
+  - freyja/variants
+  - freyja/demix
+  - freyja/update
+  - freyja/boot
+
+input:
```

Comment:
Can you update your meta file to follow the guidelines? It should contains all input and output channels/values with the structure of those channels. See [here](https://nf-co.re/docs/contributing/subworkflows#documentation) for an example